tools/x-workbench/.storybook/build-story.js

```diff
@@ -56,6 +57,11 @@ function buildStory ({package: pkg, dependencies, component: Component, knobs, s
 	storybook.add(story.title, () => {
 		const props = createProps(story.data, story.knobs, knobs);
 
+		if (story.fetchMock) {
+			fetchMock.restore(); // to isolate the mocks to each story
```

Comment:
Thinking about it more, this may cause us trouble if we have multiple stories with mocks, as they're all working with the same instance of `fetchMock`. I've not used it before but we might want to consider [`fetchMock.sandbox()`](http://www.wheresrhys.co.uk/fetch-mock/api#sandbox)